pyproject.toml

```diff
@@ -42,6 +42,7 @@ uvicorn = "*"
 numpy = "*"
 json-repair = "*"
 browsergym-core = "0.13.3"                         # integrate browsergym-core as the browsing interface
+playwright = "^1.55.0"
```

Comment:
Make explicit instead of transitive through browsergym-core so we get the upgrade can install on Debian Trixie